packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx

```diff
@@ -26,40 +24,48 @@ export default function BillingTile({ id }: TBillingTileProps) {
   const nextBillingDate = billingService?.data?.billing?.nextBillingDate;
 
   return (
-    <OsdsTile>
-      <div className="flex flex-col w-full">
-        <CommonTitle>
-          {t('managed_vcd_dashboard_service_management')}
-        </CommonTitle>
-        <OsdsDivider separator />
-        <TileSubtitle>{t('managed_vcd_dashboard_mailing_list')}</TileSubtitle>
-        <Description>{user?.email}</Description>
-        <OsdsDivider separator />
-        <TileSubtitle>{t('managed_vcd_dashboard_service_renew')}</TileSubtitle>
-        {nextBillingDate ? (
-          <Links
-            onClickReturn={() => {}}
-            label={dateTimeFormat?.format(new Date(nextBillingDate))}
-          />
-        ) : (
-          <span>-</span>
-        )}
-        <OsdsDivider separator />
-        <TileSubtitle>{t('managed_vcd_dashboard_password')}</TileSubtitle>
-        <div className="flex-wrap">
-          <OsdsLink disabled>
-            {t('managed_vcd_dashboard_password_renew')}
-          </OsdsLink>
-          <OsdsChip
-            inline
-            color={ODS_THEME_COLOR_INTENT.primary}
-            className="ml-3"
-            size={ODS_CHIP_SIZE.sm}
-          >
-            {t('managed_vcd_dashboard_coming_soon')}
-          </OsdsChip>
-        </div>
-      </div>
-    </OsdsTile>
+    <div className="h-fit">
+      <DashboardTile
+        title={t('managed_vcd_dashboard_service_management')}
+        items={[
+          {
+            id: '',
```

Comment:
Why id is always `''` ? I guess it's used for key prop so it should be set with an unique value, right ?

Reply:
Yes, it's fixed!